helm/templates/configmap.yaml

```diff
@@ -1,23 +1,45 @@
-{{- $chartName := include "ts-server-boilerplate.name" . -}}
-{{- $tracingUrl := include "ts-server-boilerplate.tracingUrl" . -}}
-{{- $metricsUrl := include "ts-server-boilerplate.metricsUrl" . -}}
+{{- $chartName := include "polygon-parts-manager.name" . -}}
+{{- $tracing := (include "common.tracing.merged" .) | fromYaml }}
+{{- $metrics := (include "common.metrics.merged" .) | fromYaml }}
+{{- $db := (include "common.db.merged" .) | fromYaml }}
 {{- if .Values.enabled -}}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ .Release.Name }}-{{ $chartName }}-configmap
 data:
+  ARRAY_SEPARATOR: {{ .Values.env.application.arraySeparator | quote }}
+  CREATE_POLYGON_PARTS_TABLES_STORED_PROCEDURE: {{ .Values.env.application.createPolygonPartsTablesStoredProcedure | quote }}
+  UPDATE_POLYGON_PARTS_TABLES_STORED_PROCEDURE: {{ .Values.env.application.updatePolygonPartsTablesStoredProcedure | quote }}
+  ENTITY_PARTS_NAME_PREFIX: {{ .Values.env.application.entities.parts.namePrefix | quote }}
+  ENTITY_PARTS_NAME_SUFFIX: {{ .Values.env.application.entities.parts.nameSuffix | quote }}
+  ENTITY_POLYGON_PARTS_NAME_PREFIX: {{ .Values.env.application.entities.polygonParts.namePrefix | quote }}
+  ENTITY_POLYGON_PARTS_NAME_SUFFIX: {{ .Values.env.application.entities.polygonParts.nameSuffix | quote }}
+  LOG_LEVEL: {{ .Values.env.logLevel | quote }}
+  LOG_PRETTY_PRINT_ENABLED: {{ .Values.env.logPrettyPrintEnabled | quote }}
+  DB_HOST: {{ $db.host }}
+  DB_PORT: {{ $db.port | quote }}
+  DB_NAME: {{ $db.name }}
+  DB_SCHEMA: {{ $db.schema }}
+  DB_LOGGING: {{ $db.logging | quote }}
+  DB_APPNAME: {{ $db.applicationName | quote }}
+  DB_ENABLE_SSL_AUTH: {{ $db.sslEnabled | quote }}
+  {{- if $db.sslEnabled }}
+  DB_CA_PATH: {{ $db.sslAuth.sslMountPath }}/{{ $db.sslAuth.caFileName }}
+  DB_KEY_PATH: {{ $db.sslAuth.sslMountPath }}/{{ $db.sslAuth.keyFileName }}
+  DB_CERT_PATH: {{ $db.sslAuth.sslMountPath }}/{{ $db.sslAuth.certFileName }}
+  {{- end }}
   REQUEST_PAYLOAD_LIMIT: {{ .Values.env.requestPayloadLimit | quote }}
   RESPONSE_COMPRESSION_ENABLED: {{ .Values.env.responseCompressionEnabled | quote }}
   LOG_LEVEL: {{ .Values.env.logLevel | quote }}
   LOG_PRETTY_PRINT_ENABLED: {{ .Values.env.logPrettyPrintEnabled | quote }}
-  {{ if .Values.env.tracing.enabled }}
-  TELEMETRY_TRACING_ENABLED: 'true'
-  TELEMETRY_TRACING_URL: {{ $tracingUrl }}
-  {{ end }}
-  {{ if .Values.env.metrics.enabled }}
-  TELEMETRY_METRICS_ENABLED: 'true'
-  TELEMETRY_METRICS_URL: {{ $metricsUrl }}
-  {{ end }}
+  TELEMETRY_TRACING_ENABLED: {{ $tracing.enabled | quote }}
+  {{- if $tracing.enabled }}
+  TELEMETRY_TRACING_URL: {{ $tracing.url }}
+  {{- end }}
+  TELEMETRY_METRICS_ENABLED: {{ $metrics.enabled | quote }}
+  {{- if $metrics.enabled }}
+  TELEMETRY_METRICS_URL: {{ $metrics.url }}
```

Comment:
I see that  `TELEMETRY_METRICS_INTERVAL` is not included, should it be?

Reply:
adding